test/acceptance/build/github_test.js

```diff
@@ -324,6 +336,61 @@ describe('Acceptance | Build | Github', function () {
         expect(res.statusCode).to.equal(200);
         expect(res.result).to.eql('RA disabled for this PR');
       });
+
+      describe('when RA does not already exist', function () {
+        it('responds with 200 and creates RA if it does not exist', async function () {
```

Comment:
responds with 200 and creates missing RA if needed

Reply:
Modifié, on donne déjà le contexte au dessus dans le describe, c'était too much :p